frontend/dashboard/pages/PageLayout/DashboardHeader/SmallHeaderMenu/SmallHeaderMenu.tsx

```diff
@@ -0,0 +1,105 @@
+import React, { useState } from 'react';
+import type { ReactElement } from 'react';
+import classes from './SmallHeaderMenu.module.css';
+import { useTranslation } from 'react-i18next';
+import { StudioAvatar, StudioButton, StudioParagraph } from '@studio/components';
+import { MenuHamburgerIcon } from '@studio/icons';
+import { DropdownMenu } from '@digdir/designsystemet-react';
+import { useHeaderContext } from '../../../../context/HeaderContext';
+import type { HeaderMenuGroup } from '../../../../types/HeaderMenuGroup';
+import { SmallHeaderMenuItem } from './SmallHeaderMenuItem';
+import type { NavigationMenuItem } from '../../../../types/NavigationMenuItem';
+import type { NavigationMenuGroup } from '../../../../types/NavigationMenuGroup';
+import {
+  groupMenuItemsByGroup,
+  mapHeaderMenuGroupToNavigationMenu,
+} from '../../../../utils/headerUtils/headerUtils';
```

Comment:
Import can be shortened as the utils folder has a index.ts file
```suggestion
import {
  groupMenuItemsByGroup,
  mapHeaderMenuGroupToNavigationMenu,
} from '../../../../utils/headerUtils';
```